nengo_gui/namefinder.py

```diff
@@ -22,9 +22,12 @@ def find_names(self, net):
         classes = (nengo.Node, nengo.Ensemble, nengo.Network,
                    nengo.Connection)
 
+        private = False
+        in_lists = False
```

Comment:
> Because it's bad to initialise variables inside a loop? I can't find a reference for this however...

Hmm, i don't think I share that intuition.  Mostly, for me, it's about clarity -- when I read this code I got confused looking for the reason why it was initialized like that, especially for a variable that's only used inside the loop.  Since it was defined outside it looked like you needed a default value for if the loop executed zero times....